src/Draco.Compiler/Syntax/CSharpTranspiler.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Draco.Compiler.Utilities;
+using static Draco.Compiler.Syntax.ParseTree;
+
+namespace Draco.Compiler.Syntax;
+internal sealed class CSharpTranspiler : ParseTreeVisitorBase<Unit>
+{
+    public override Unit Default => new Unit();
+    public StringBuilder output = new StringBuilder();
```

Comment:
You should not expose a raw field like this. Maybe keep this as a `private readonly` and then have a public property that does `public string GeneratedCode => this.stringBuilder.ToString()`